src/extensions/scratch3_goforce/index.js

```diff
@@ -0,0 +1,202 @@
+
+const CompareOptionsMenu = ['>', '<', '='];
+
+const DirectionOptionsMenu = ['x', 'y', 'z'];
+
+
+/**
+ * Scratch 3.0 blocks to interact with a GoForce peripheral.
+ */
+class Scratch3GoForceBlocks {
+
+    /**
+     * @return {string} - the name of this extension.
+     */
+    static get EXTENSION_NAME () {
+        return 'Go! Force';
```

Comment:
We spoke about naming conventions a little and agreed that, while a bit terse, GDX-FOR is the order code and is printed on the label. We have an older product line that used the 'Go!' prefix, so that could be confusing. So, how about using 'gdx-for' in the source code, and 'GDX-FOR' for user facing strings.

Reply:
ok, I changed the user facing strings to "GDX-FOR". In the code, the extension ID is "gdxfor", without the "-", because we can't have "-" in variable names. Is that alright with you @stocktonkincade?

Reply:
Absolutely